src/containers/EventCreationPage.tsx

```diff
@@ -82,24 +102,59 @@ const EventCreationPage = () => {
     : {};
 
   const handleComplete = () => {
-    addEvent({
-      variables: {
-        name: eventName,
-        eventDate:
-          dateParts.year &&
-          dateParts.month &&
-          dateParts.day &&
-          `${dateParts.year}-${dateParts.month.padStart(
-            2,
-            '0'
-          )}-${dateParts.day.padStart(2, '0')}`,
-        createdBy: 1,
-        isActive: true,
-      },
-    });
+    if (mode === 'new') {
+      addEvent({
+        variables: {
+          name: eventName,
+          eventDate:
+            dateParts.year &&
+            dateParts.month &&
+            dateParts.day &&
+            `${dateParts.year}-${dateParts.month.padStart(
+              2,
+              '0'
+            )}-${dateParts.day.padStart(2, '0')}`,
+          createdBy: 16, // TODO: change this to proper user
```

Comment:
this needs to be changed back to 1 for now - all dev envs and staging guaranteed has userid 1 but not 16
```suggestion
          createdBy: 1, // TODO: change this to proper user
```

Reply:
good call, will make this fix